fastlane/lib/fastlane/actions/verify_build.rb

```diff
@@ -66,7 +66,10 @@ def self.gather_cert_info(app_path)
       end
 
       def self.update_with_profile_info(app_path, values)
-        profile = `cat #{app_path.shellescape}/embedded.mobileprovision | security cms -D`
+        provision_profile_path = "#{app_path}/embedded.mobileprovision"
+        UI.user_error!("Unable to extract profile") unless File.exist?(provision_profile_path)
```

Comment:
```suggestion
        UI.user_error!("Unable to find embedded profile in #{provision_profile_path}") unless File.exist?(provision_profile_path)
```

Reply:
I guess I will have to readjust the tests for that.